plugin.json

```diff
@@ -57,6 +57,12 @@
         "display_name": "Application ID",
         "type": "text",
         "help_text": "Application ID from the Azure Active Directory. Only required for Skype for Business Online. Use this URL to configure the plugin in Azure Active Directory: ```https://SITEURL/plugins/skype4business/api/v1/auth_redirect```"
+      },
+      {
+        "key": "ShouldLogClientRequests",
```

Comment:
Why is a separate config value needed? Is logging on a debug level enough?

Reply:
@hanzei we were talking with @mickmister about this in the [issue](https://github.com/mattermost/mattermost-plugin-skype4business/issues/58) page. Basically, we think, that logging all requests in S4B would create unnecessary spam in the logs, so it's better to make this feature optional. What are your thoughts?

Reply:
I'm 1/5 that a separate config value is not worth it if there is already a building tool (debug logs) in Mattermost. @mickmister What makes you think that a separate config value is needed?

Reply:
@hanzei Makes sense to me to just log via debug actually. Is there any drawback to unconditionally throw these raw requests to `API.LogDebug`, in the case that the log level is not set to `DEBUG`?

Reply:
Well, there might be a performance concern if we call an `RPC` method every time `API.LogDebug` one called. But Incoming http request should happen /that/ often and hence I think it's fine to leverage `API.LogDebug`.